From ba55985a050750483553584ebf6b587f4e6b7f91 Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date: Thu, 4 Jul 2019 11:52:43 +0100
Subject: [PATCH 695/703] configs: Drop MMC_SDHCI_BCM2711 from
 arm64/bcm2711_defconfig

Apparently this is a vestigial setting and should be removed.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>
---
 arch/arm64/configs/bcm2711_defconfig | 1 -
 1 file changed, 1 deletion(-)

--- a/arch/arm64/configs/bcm2711_defconfig
+++ b/arch/arm64/configs/bcm2711_defconfig
@@ -1031,7 +1031,6 @@ CONFIG_USB_G_HID=m
 CONFIG_USB_G_WEBCAM=m
 CONFIG_MMC=y
 CONFIG_MMC_BLOCK_MINORS=32
-CONFIG_MMC_SDHCI_BCM2711=y
 CONFIG_MMC_BCM2835_MMC=y
 CONFIG_MMC_BCM2835_DMA=y
 CONFIG_MMC_BCM2835_SDHOST=y
